cmd/humioctl/views_list.go

```diff
@@ -31,10 +35,18 @@ func newViewsListCmd() *cobra.Command {
 
 			rows := make([][]format.Value, len(views))
 			for i, view := range views {
-				rows[i] = []format.Value{format.String(view.Name)}
+				if viewOnly && view.Typename == viewTypeName {
+					rows[i] = []format.Value{format.String(view.Name)}
+				} else {
+					rows[i] = []format.Value{format.String(view.Name)}
```

Comment:
I don't think this makes sense. The "if" and "else" seems to be doing the exact same thing, so the flag doesn't really do anything in the current implementation as the result is the same.

Reply:
You're absolutely correct. I have no idea what I was doing when I pushed this.